templates/bq_elasticc_SuperNNova_schema.json

```diff
@@ -28,5 +28,11 @@
       "mode": "NULLABLE",
       "name": "predicted_class",
       "type": "INTEGER"
+    },
+    {
+      "description": "timestamp for testing",
+      "mode": "REQUIRED",
```

Comment:
I would make this `NULLABLE` not `REQUIRED`. No reason to prevent the rest of the data in the row from being added if the timestamp happens to be missing, IMO. But this is a matter of taste and not deeply important.

Reply:
I hadn't thought about that, but I agree with you. Will update